README.txt

```diff
@@ -1,207 +1,186 @@
-#----------End of documentation for DATA_STORAGE----------#  
-
-#----------MIN_CIRCLE----------#
-# Function defining the inside concentric circle of the ring of propagation of a call
-# and checking if a specific point is outside the circle. Returns a boolean.
-# NB: x and y have to be int or np.arrays for this function to work
-# Inputs: center_x, center_y, radius, x, y 
-# center_x: integer. Abscissa of the center of the circle (i.e. source)
-# center_y: integer. Ordinate of the center of the circle (i.e. source)
-# radius: integer. radius of the circle
-# x: integer. Abscissa of the point to be checked for
-# y: integer. Ordinate of the point to be checked for
+# dict1: dictionary. Contains the call to be checked for
+# tcall: time at which the call was emitted
 
 #----------OUTPUTS----------#
-# dist: integer. Distance of the specific point to the center of the circle.
-# If dist is more than radius, the function returns TRUE.
-#----------End of documentation for MIN_CIRCLE----------#
-
-#----------MAX_CIRCLE----------#
-# Function defining the outside concentric circle of the ring of propagation of a call
-# and checking if a specific point is inside the circle. Returns a boolean.
-# NB: x and y have to be int or np.arrays for this function to work
-# Inputs: center_x, center_y, radius, x, y 
-# center_x: integer. Abscissa of the center of the circle (i.e. source)
-# center_y: integer. Ordinate of the center of the circle (i.e. source)
-# radius: integer. radius of the circle
-# x: integer. Abscissa of the point to be checked for
-# y: integer. Ordinate of the point to be checked for
+# dict1: Updated dict1 dictionary, where the call has been erased if too old
+#----------End of Documentation for Data_storage----------#
 
-#----------OUTPUTS----------#
-# dist: integer. Distance of the specific point to the center of the circle.
-# If dist is less than radius, the function returns TRUE.
-#----------End of documentation for MAX_CIRCLE----------#
+#----------Propagation----------#
+# Function that increments the distance of propagation of a call with time
+# Inputs:  dict1, tmstp
+# dict1: dictionary. Contains the distance information to be updated
+# tmstp: time at which the call was emitted
 
-#----------PROPAGATION----------#
-# Function for calculation of the distance of the sound from its source as it propagates
-# Inputs taken from __init__ method: ID, timestep
-# Inputs: dict1
-# Other inputs: timestore taken from data_storage method
-# dict1: dictionary (here, callsource) from which the propagation distance is extracted
+#----------OUTPUTS----------#
+# dict1: Updated dict1 dictionary, with the incremented value of propagation distance
+#----------End of Documentation for Propagation----------#
+
+#----------IN_RING----------#
+# Function that tests whether a certain point (agent) is on the ring of sound.
+# Inputs: center_x, center_y, radius, x, y
+# center_x: float. Abscissa of the source of the call
```

Comment:
center_x is not so informative - could you change name to:
callsource_x and callsource_y